recipes-core/packagegroups/packagegroup-xenclient-installer.bb

```diff
@@ -22,6 +23,7 @@ RDEPENDS_${PN} = " \
     xen-libxlutil \
     xen-xenstat \
     xen-xl \
+    xen-xenstored-ocaml \
```

Comment:
Why xenstored now needed?

Reply:
Strictly speaking it is not needed but the build was complaining about missing dependencies otherwise. I suspect this was needed beforehand but just happened to work because the installer shared the same image with dom0.